notebooks/SITCOMTN_101_FCU_investigation.ipynb

```diff
@@ -0,0 +1,1216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Check whether the TMA stayed at the same azimuth and elevation during the vibration measurements
</pre><p>Instead of having these comments as code comments, I recommend you add them in the previous cell as markdown text. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-101/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
done